ruby/.github/workflows/tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           ruby-version: 2.6.x
       - name: Setup RSpec
         run: |
-          [ -f Gemfile ] && bundle --deployment
+          [ -f Gemfile ] && bundle
```

Comment:
It seems this is the only place we used the `--deployment` flag

Reply:
oh nice!